lhotse/cut.py

```diff
@@ -36,6 +35,14 @@ class Cut:
     # cut duration. They also might overlap.
     supervisions: List[SupervisionSegment]
```

Comment:
maybe document how you would get the feature num-frames and dimension, i.e. the shape of the features?
Would be useful to know at this point, for a typical user.

Reply:
Great idea! While writing the example script and notebook, I noticed that the CLI is actually nicer to use at this point than the Python lib, which is quite low-level for now. I have some ideas on how to build easier-to-use higher-level APIs on top of what's there, but I want to add a bit more recipes and task types (e.g. ASR) to get a better understanding first.

Reply:
I'll actually need to make some changes to the `Features` object address this cleanly. I updated the issue #20 with details, and I'll take care of this soon after this PR.